avformat/avformat.go

```diff
@@ -81,10 +84,10 @@ func (ctx *Context) SetInterruptCallback() {
 type Flags int
 
 const (
-	FlagNoFile       Flags = C.AVFMT_NOFILE
-	FlagNeedNumber   Flags = C.AVFMT_NEEDNUMBER
-	FlagShowIDs      Flags = C.AVFMT_SHOW_IDS
-	FlagRawPicture   Flags = C.AVFMT_RAWPICTURE
+	FlagNoFile     Flags = C.AVFMT_NOFILE
+	FlagNeedNumber Flags = C.AVFMT_NEEDNUMBER
+	FlagShowIDs    Flags = C.AVFMT_SHOW_IDS
+	// FlagRawPicture   Flags = C.AVFMT_RAWPICTURE
```

Comment:
Is implementing this a TODO? If so, can it be marked as such? If not, can it be removed?

Reply:
👍  Removed